orangecontrib/text/widgets/tests/test_owldavis.py

```diff
@@ -96,11 +97,21 @@ def test_relevance_changed(self):
             self._get_graph_labels()[:6],
         )
 
-    def test_report(self):
-        self.send_signal(self.widget.Inputs.topics, self.topics)
+    @patch("orangecontrib.text.widgets.owldavis.OWLDAvis.report_items")
+    @patch("orangecontrib.text.widgets.owldavis.OWLDAvis.report_plot")
+    def test_report(self, mocked_plot, mocked_items: Mock):
+        self.send_signal(self.widget.Inputs.topics, None)
         self.wait_until_finished()
+        self.widget.send_report()
+        mocked_items.assert_called_once()
+        mocked_plot.assert_not_called()
+        mocked_items.reset_mock()
 
+        self.send_signal(self.widget.Inputs.topics, self.topics)
+        self.wait_until_finished()
         self.widget.send_report()
+        mocked_items.assert_called_once()
+        mocked_plot.assert_not_called()
```

Comment:
I think this one should be:
 
```suggestion
        mocked_plot. assert_called_once()
```